### DISCUSSION TITLE
NotFound http error when calling Olarm API

### DISCUSSION THREAD

**Discussion by rainepretorius** [rainepretorius/olarm-ha-integration#9] [Announcements]
Just so you know, if you are receiving a NotFound or APINotFound error. It means that the entered device id is incorrect, and Olarm's API generates this error. Please double-check your device id before opening an issue.

**Comment by pawley84** [comment]
Hi Raine,

Thanks for starting this project. I've been waiting to integrate my alarm for ages.

I am getting the 'Not Found' after installing and following the steps. My device ID also ends in a char and not a numeric... "73b81366-ebcb-MASK-9143-MASKED373e3e".

I have an olarm pro with wifi and gsm. My alarm type is DSC.

Please let me know if there are any troubleshooting steps I should follow or if there are any other details that would help.

Thank you
Matt

**Reply by rainepretorius** [reply]
Hi Pawley,
Can you send me the response of this url:
https://apiv4.olarm.co/api/v4/devices/your_device_id?accessToken=your_api_key to raine.pretorius1@gmail.com. It can be something as simple as a toggle not being switched on in Olarm's app.

**Reply by pawley84** [reply]
Thank you for your prompt response.

I've just emailed you the full response that I get from that link.

**Reply by rainepretorius** [reply]
Thanks. I will have a look at it.
